### DISCUSSION TITLE
XCT localization Resx Files

### DISCUSSION THREAD

**Discussion by Orgbrat** [xamarin/XamarinCommunityToolkit#1206] [Q&A] [answered]
So when adding the Resx language files to the solution in the .Net Standard project is there a particular location they should be added.? They should be named AppResources.resx and then AppResources.xx.resx for each language is what I understand. 
But do they have to be in a certain folder for the LocalizationResourceManager to be able to locate them? 
Or how does the LocalizationResourceManager locate and load the file?

Orgbrat

**Comment by pictos** [comment]
@Orgbrat AFAIK you can put that file anywhere on your project. I'll ping @maxkoshevoi to double-check

**Comment by maxkoshevoi** [comment, marked as answer]
@pictos is correct. They can be added anywhere, and have any name.
Here's how they look in my pet project for example: https://github.com/maxkoshevoi/NureTimetable/tree/main/NureTimetable.Core/Localization

You can see that they are named `LN` and located in completely different project from UI stuff.

The only rule is that if your default resource file is named `X.resx`, resource files for other languages must be in the same folder and named `X.yy.resx` or `X.yy-zz.resx`. 
For example: `X.en.resx` or `X.en-US.resx`

**Reply by Orgbrat** [reply]
Thanks everybody for the great information...

**Reply by maxkoshevoi** [reply]
Also here's more info on localization: https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/localization/text
